lua/terrortown/lang/en.lua

```diff
@@ -2304,3 +2304,36 @@ L.label_button_level_reset = "reset level"
 L.loadingscreen_round_restart_title = "Starting new round"
 L.loadingscreen_round_restart_subtitle = "you're playing on {map}"
 L.loadingscreen_round_restart_subtitle_limits = "you're playing on {map} for another {rounds} round(s) or {time}"
+
+-- 2024-06-23
+L.help_roles_derandomize = [[
+Role derandomization can be used to make role distribution feel more fair over the course of a session.
```

Comment:
I think we should maybe try to find some better wording than derandomizing, maybe pseudo-random?
But idk, would also be fine its just somewhat irritating for players.

@TimGoll do you have an idea on what to call this?

Reply:
In general you call it "weighted random" afaik. So maybe we should lean into that. Something like "Weighted role selection can be used..."

Reply:
I feel like "weighted" suggests something that is more directly configurable, rather than a mostly automatic. My first thought when seeing "weighted role selection" would be a mode where I (as a server owner) can adjust weights applied to each role to make them more or less likely to appear. While this is *implemented* using weights (and that is how the probabilities are configured), conceptually it is just "reduce the probability of receiving a role according to how recently you've had that role". Hence, "derandomization".

Reply:
I get what you mean, let me think about it